### PR TITLE
Cnn baseline result with augmented

### PR DESCRIPTION
**Pull request by NevinJulian**

### CODE REVIEW

File: notebooks/07-bird-whisperer-visualize-scores.ipynb
Comment (Davee02) [inline notebooks/07-bird-whisperer-visualize-scores.ipynb:1]: revert this change as it would likely results in a merge conflict

File: notebooks/cnn_baseline/experiment_utils.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        raise argparse.ArgumentTypeError(f"readable_dir:{path} is not a valid path")
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Bird Whisperer Trainer")
+
+    parser.add_argument("--seed", type=int, default=42, help="Seed for reproducibility (default: 42)")
+    parser.add_argument("--lr", type=float, default=3e-4, help="Learning rate (default: 3e-4)")
+    parser.add_argument("--epochs", type=int, default=10, help="Number of epochs (default: 10)")
+    parser.add_argument("--with_augmented", action="store_true", default=False, help="Use augmented data (default: True)")
+    parser.add_argument("--whisper_base_variant", type=str, default="tiny", help="Whisper model base variant (default: tiny)")
```
Comment (Davee02) [inline notebooks/cnn_baseline/experiment_utils.py:38]: remove this parameter as you don't need it

File: notebooks/cnn_baseline/models/improved_cnn.py
Comment (Davee02) [inline notebooks/cnn_baseline/models/improved_cnn.py:1]: do you still need this file?

**Review by Davee02 — CHANGES_REQUESTED** [review]

File: notebooks/cnn_baseline/models/improved_cnn.py
Comment (NevinJulian) [reply notebooks/cnn_baseline/models/improved_cnn.py:1]: This was my first test before using the correct utils that are also used for the bird whisperer, I will move it to the _old folder

**Review by Davee02 — APPROVED** [review]
